CHANGELOG.md

```diff
@@ -124,6 +128,106 @@ Cortex 0.4.0 is the last version that can *write* denormalised tokens. Cortex 0.
 
 Cortex 0.6.0 is the last version that can *read* denormalised tokens. Starting with Cortex 0.7.0 only normalised tokens are supported, and ingesters writing denormalised tokens to the ring (running Cortex 0.4.0 or earlier with `-ingester.normalise-tokens=false`) are ignored by distributors. Such ingesters should either switch to using normalised tokens, or be upgraded to Cortex 0.5.0 or later.
 
+### Known issues
+
+- The gRPC streaming for ingesters doesn't work when using the experimental TSDB blocks storage. Please do not enable `-querier.ingester-streaming` if you're using the TSDB blocks storage.
```

Comment:
👍 Perhaps we can mention that it should be fixed in master now, but being an experimental feature, it didn't make it to 0.7.0.